pkg/3scale/amp/component/backend.go

```diff
@@ -387,6 +398,10 @@ func (backend *Backend) buildBackendWorkerEnv() []v1.EnvVar {
 	result = append(result,
 		helper.EnvVarFromSecret("CONFIG_EVENTS_HOOK", "system-events-hook", "URL"),
 		helper.EnvVarFromSecret("CONFIG_EVENTS_HOOK_SHARED_SECRET", "system-events-hook", "PASSWORD"),
+		// TODO: Add as EnvVar or reference to CM?
+		// TODO: Enable metrics endpoint only when monitoring is enabled by CR? Other components have monitoring enabled no matter monitoring field is enabled or not.
```

Comment:
Seems good to me enabled by default (unless there's a considerably noticeable performance hit doing so). As far as I know for APIcast for example does the same.

Reply:
:+1: I will remove comments, then.